src/qt/dbb_gui.cpp

```diff
@@ -644,7 +644,7 @@ void DBBDaemonGui::showEchoVerification(DBBWallet* wallet, const UniValue& propo
 {
 
     int amountOfClientsInformed = 0;
-    if (comServer)
+    if (comServer && !comServer->getChannelID().empty())
```

Comment:
Doesn't work because the channel ID is independent of whether or not the mobile app is connected to the desktop app. Need to set a variable depending if the 'ping' command returns a 'pong' or not.